backend/backend.go

```diff
@@ -555,6 +564,42 @@ func (backend *Backend) Coin(code coinpkg.Code) (coinpkg.Coin, error) {
 	return coin, nil
 }
 
+func (backend *Backend) pollETHBalances() {
+	timer := time.After(0)
+
+	for {
+		select {
+		case <-backend.quit:
```

Comment:
Should do a nested check for this like in https://github.com/BitBoxSwiss/block-client-go/pull/8/files#diff-8d7b38d7d1da47b2e7c4b04453a94761e959c337444e1aa135dcaa88ac0f4392R174

The reason is that if timer and quit are available at the same time, you want to prioritize quit.